src/main/scala/org/hammerlab/guacamole/commands/SomaticStandardCaller.scala

```diff
@@ -113,6 +117,28 @@ object SomaticStandard {
           args.minTumorAlternateReadDepth
         )
 
+      if (args.dbSnpVcf != "") {
+        val adamContext = new ADAMContext(sc)
+        val dbSnpVariants = adamContext.loadVariantAnnotations(args.dbSnpVcf)
+        potentialGenotypes
```

Comment:
You're right! This should have been `potentialGenotypes = ...` I couldn't get this into the output, since that needs to come in ADAM, but I should fix this.